### PR TITLE
Adds a new core deepmaint room, Chrome Labs

### PR DESCRIPTION
**Pull request by bearrrrrrrr**
## About The Pull Request

Adds a new core deepmaint room, properly modularized under occulus' files. Pictures are below. Uses name 'chrome laboratory' in code.

Don't go peeking unless you want to be spoiled.

## Changelog
```changelog
add: new deepmaint room; chrome laboratory.
```

### CODE REVIEW

**Comment by bearrrrrrrr** [comment]
Don't go peeking if you want spoilers.

image

**Comment by bearrrrrrrr** [comment]
Btw, tested re: open vents, and open vents don't make the vent sound effect in deepmaint, presumably b/c there are no pipes to actually push air through due to deepmaint...not spawning pipes.

File: zzzz_modular_occulus/maps/submaps/deepmaint_rooms/core/core_rooms.dm [hunk 1]
```diff
@@ -0,0 +1,4 @@
+/datum/map_template/deepmaint_template/big/podbay
```
Comment (Michiyamenotehifunana) [inline zzzz_modular_occulus/maps/submaps/deepmaint_rooms/core/core_rooms.dm:1]: Is this quite right?

**Review by Michiyamenotehifunana — CHANGES_REQUESTED** [review]
Looks good except for some copy pasta you forgot to change

File: zzzz_modular_occulus/maps/submaps/deepmaint_rooms/core/core_rooms.dm (hunk 1 above)
Comment (bearrrrrrrr) [reply zzzz_modular_occulus/maps/submaps/deepmaint_rooms/core/core_rooms.dm:1]: O O P S

**Review by Michiyamenotehifunana — APPROVED** [review]
All good!

**Pull request merged**